packages/components/src/components/Tooltip/TooltipBox.tsx

```diff
@@ -0,0 +1,109 @@
+import { Icon, IconType } from '../assets/Icon/Icon';
+import { ReactElement, ReactNode } from 'react';
+import { motion } from 'framer-motion';
+import { borders, palette, spacings, spacingsPx, typography } from '@trezor/theme';
+import styled from 'styled-components';
+
+export const TOOLTIP_BORDER_RADIUS = borders.radii.sm;
+
+const getContainerPadding = (isLarge: boolean, isWithHeader: boolean) => {
+    if (isLarge) {
+        if (isWithHeader) {
+            return `${spacingsPx.sm} ${spacingsPx.md} ${spacingsPx.xs}`;
+        }
+
+        return `${spacingsPx.xs} ${spacingsPx.md}`;
+    }
+
+    return spacingsPx.xs;
+};
+
+type TooltipContainerStyledProps = {
+    $maxWidth: string | number;
+    $isLarge: boolean;
+    $isWithHeader: boolean;
+};
+
+const TooltipContainerStyled = styled(motion.div)<TooltipContainerStyledProps>`
+    background: ${palette.darkGray300};
+    color: ${palette.lightWhiteAlpha1000};
+    border-radius: ${TOOLTIP_BORDER_RADIUS};
+    text-align: left;
+    border: solid 1.5px ${palette.darkGray100};
+    max-width: ${props => props.$maxWidth}px;
+    ${typography.hint}
+
+    > div {
+        padding: ${({ $isLarge, $isWithHeader }) => getContainerPadding($isLarge, $isWithHeader)};
+    }
+`;
+
+const HeaderContainer = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: ${spacingsPx.sm};
+    width: 100%;
+`;
+
+const TooltipTitle = styled.div<{ $isLarge: boolean }>`
+    display: flex;
+    align-items: center;
+    gap: ${spacingsPx.xxs};
+    width: 100%;
+    color: ${({ theme }) => theme.textSubdued};
+    ${({ $isLarge }) => ($isLarge ? typography.highlight : typography.hint)}
+`;
+
+const Addon = styled.div`
+    margin-left: auto;
+`;
+
+export type TooltipBoxProps = {
+    content: ReactNode;
+    maxWidth?: string | number;
+    /**
+     *  @description Legacy prop
```

Comment:
shouldn't it be deprecated?

Reply:
I know, just thinking :) if you IDE strikethrough the text it could avoid future usage.

Reply:
One practice I like to do is to rename the deprecated field to something like `isLargeDeprecated`

Reply:
But it needs broader refactoring.From my point of view, mark is as `@deprecated` is good enough.

Reply:
deprecated added in 59fd674a1e 

Sorry I was not reading carefully and I thought there is `@deprecated` all the time, so I got confused :D